### PR TITLE
Product page design

### PR DESCRIPTION
**Pull request by kkellydesign**
This PR is branched from `firefox-accounts-integration, and covers the following: 
* “Ask the Community” and “Contact Us” are now larger callouts, taken from the Zeplin screens.
* Logo/Title area design edits have been implemented, and ‘Frequent Topics’ headline/copy have been added
* Page width has been expanded with a modifier class on the body tag for that page. We’ll be able to apply that as we go if it’s needed elsewhere.
* This does NOT touch the navigation. 
* No changes have been made to the mobile codebase.

ref #3531

### CODE REVIEW

File: kitsune/products/jinja2/products/product.html [hunk 1]
```diff
@@ -11,15 +11,34 @@
 
 {% block title %}{{ _('{product} Help')|f(product=pgettext('DB: products.Product.title', product.title)) }}{% endblock %}
 
+{% block base_container_classes %}container_wide{% endblock %}
+
 {% block content %}
-  <div class="grid_12 search-form-large">
+<header class="help-topics-header">
+      <h1 class="product-title">
+        <img class="product-logo" src="{{ product.image_alternate_url }}" alt="" />
```
Comment (mcranston18) [inline kitsune/products/jinja2/products/product.html]: Could we add an `alt` tag for this image? e.g. `Photo of {{ product.title }}`

Comment (kkellydesign) [reply kitsune/products/jinja2/products/product.html]: Sure – fixed.

File: kitsune/products/jinja2/products/product.html [hunk 2]
```diff
@@ -11,15 +11,34 @@
 
 {% block title %}{{ _('{product} Help')|f(product=pgettext('DB: products.Product.title', product.title)) }}{% endblock %}
 
+{% block base_container_classes %}container_wide{% endblock %}
+
 {% block content %}
-  <div class="grid_12 search-form-large">
+<header class="help-topics-header">
```
Comment (johngian) [inline kitsune/products/jinja2/products/product.html]: Please fix identation here.

**Comment by johngian** [comment]
Hey @kkellydesign i added a minor change request please update the PR and i can merge.

File: kitsune/products/jinja2/products/product.html (hunk 2 above)
Comment (mcranston18) [reply kitsune/products/jinja2/products/product.html]: @johngian done

**Pull request merged**